src/sdk/pynni/nni/smac_tuner/convert_ss_to_scenario.py

```diff
@@ -63,7 +64,8 @@ def generate_pcs(nni_search_space_content):
                                 json.dumps(search_space[key]['_value']),
                                 json.dumps(search_space[key]['_value'][0])))
                         elif search_space[key]['_type'] == 'loguniform':
-                            pcs_fd.write('%s real %s [%s] log\n' % (
+                            search_space[key]['_value'] = list(np.round(np.log(search_space[key]['_value']), 10))
```

Comment:
would be better to briefly explain why you make this change.

Reply:
I mean np.round

Reply:
okay, I will add some comments